specification/2.0/Specification.adoc

```diff
@@ -313,18 +493,12 @@ All angles are in radians.
 
 Positive rotation is counterclockwise.
 
-The <<transformations,node transformations>> and <<animations,animation channel paths>> are 3D vectors or quaternions with the following data types and semantics:
-
-* translation: A 3D vector containing the translation along the x, y and z axes
-* rotation: A quaternion (x, y, z, w), where w is the scalar
-* scale: A 3D vector containing the scaling factors along the x, y and z axes
-
-RGB color values use sRGB color primaries.
+RGB color values **MUST** use <<srgb,sRGB>> color primaries.
```

Comment:
Which RGB color values — outside of textures — use sRGB color primaries? baseColorFactor and emissiveFactor are linear, I'm not sure if those are considered "color values" even though they are factors, but I think this line may be ambiguous.

Reply:
Color primaries (aka chromaticities) have nothing to do with linear / non-linear encoding. They are essentially "units" of color values.

Factors are unitless multipliers for texture values; when the texture is missing, it's defined as having all values as `1.0`.

Reply:
What is an example of an RGB color value in the glTF specification? I would guess most readers of this line will not know the distinction between color primaries and encoding. Would this be a fair clarification?

> RGB color values **MUST** use <<srgb,sRGB>> color primaries and linear encoding, excluding textures  as described in Section XX.XX.XX.

... is it correct to say that `baseColorFactor` and `COLOR_0`, and `KHR_lights_punctual`'s `light.color` are all sRGB primaries with linear encoding?

Reply:
Completely reworded that part to avoid confusing readers with sRGB.